app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagServiceEndpointV1.java

```diff
@@ -24,13 +23,18 @@
  * Created by strubbl on 11.04.16.
  */
 public class WallabagServiceEndpointV1 extends WallabagServiceEndpoint {
+
+    public static final String WALLABAG_LOGIN_FORM_V1 = "<form method=\"post\" action=\"?login\" name=\"loginform\">";
+    public static final String WALLABAG_LOGOUT_LINK_V1 = "href=\"./?logout\"";
```

Comment:
good idea to move that constants to the corresponding service version. now we have all data in one place